src/app/policies/components/PolicyList.vue

```diff
@@ -0,0 +1,288 @@
+      <div class="stack">
+        <KCard>
+          <template #body>
+            <div class="description">
+              <div class="description-content">
+                <h3>
+                  <PolicyTypeTag :policy-type="policyType.name">
+                    {{ t('policies.collection.title', { name: policyType.name }) }}
+                  </PolicyTypeTag>
+                </h3>
+
+                <p>{{ t('policies.collection.description') }}</p>
+              </div>
+
+              <div class="description-actions">
+                <KBadge
+                  v-if="policyType.isExperimental"
+                  appearance="warning"
+                >
+                  {{ t('policies.collection.beta') }}
+                </KBadge>
+
+                <KBadge
+                  v-if="policyType.isInbound"
+                  appearance="neutral"
+                >
+                  {{ t('policies.collection.inbound') }}
+                </KBadge>
+
+                <KBadge
+                  v-if="policyType.isOutbound"
+                  appearance="neutral"
+                >
+                  {{ t('policies.collection.outbound') }}
+                </KBadge>
+
+                <DocumentationLink
+                  :href="t('policies.href.docs', { name: policyType.name })"
+                  data-testid="policy-documentation-link"
+                />
+              </div>
+            </div>
+          </template>
+        </KCard>
+
+        <KCard>
+          <template #body>
+            <AppCollection
+              class="policy-collection"
+              data-testid="policy-collection"
+              :empty-state-title="t('common.emptyState.title')"
+              :empty-state-message="t('common.emptyState.message', { type: `${policyType.name} policies` })"
+              :headers="[
+                { label: 'Name', key: 'name' },
+                policyType.isTargetRefBased ? { label: 'Target ref', key: 'targetRef' } : undefined,
+                { label: 'Actions', key: 'actions', hideLabel: true },
+              ].filter(notEmpty)"
+              :page-number="props.pageNumber"
+              :page-size="props.pageSize"
+              :total="props.policyCollection?.total"
+              :items="props.policyCollection?.items"
+              :error="props.policyError"
+              @change="emit('change', $event)"
+            >
+              <template #name="{ rowValue }">
+                <RouterLink
+                  :to="{
+                    name: 'policy-detail-view',
+                    params: {
+                      mesh: route.params.mesh,
+                      policyPath: policyType.path,
+                      policy: rowValue,
+                    },
+                  }"
+                >
+                  {{ rowValue }}
+                </RouterLink>
+              </template>
+
+              <template #targetRef="{ row }">
+                <template v-if="row.spec?.targetRef">
```

Comment:
```suggestion
                <template v-if="policyType.isTargetRefBase">
```

This looks more straight forward to me.

Reply:
Changed.